MaterialLib/MPL/Properties/CapillaryPressureSaturation/CapillaryPressureVanGenuchten.cpp

```diff
@@ -96,12 +104,8 @@ PropertyDataType CapillaryPressureVanGenuchten::value(
         return 0;
     }
 
-    double const S_eff = (S_L - _S_L_res) / (_S_L_max - _S_L_res);
-    assert(0 <= S_eff && S_eff <= 1);
-
     double const p_cap =
-        _p_b * std::pow(std::pow(S_eff, -1.0 / _m) - 1.0, 1.0 - _m);
-    assert(p_cap > 0);
```

Comment:
Removed this assertion because it is unnecessary. The formula always gives a positive value with the restrictions of all parameters in ctor.